### ISSUE TITLE
NTR, pulmo: lobar overinflation

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6436]
**Preferred term label:**
lobar overinflation

**Synonyms**
Congenital lobar overinflation, congenital lobar emphysema

**Definition (free text, please give PubMed ID)**
it is characterized by an overdistended segment of lung, it can affect an party of a lobe or the entire one. It results in progressive overinflation of one or more lobes. 

Comment: More commonly present in neonate or early infancy with life-threatening respiratory distress. The upper lobes are more frequently affected than the lower one. And more common in males (M:F = 3:1) 

On imaging, it classically presents on chest radiographs as a hyperlucent lung segment with overinflation and contralateral mediastinal shift. 

**Parent term (use hpo.jax.org/app)**
Emphysema HP:0002097; Abnormal lung lobation HP:0002101

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:1928   (Disorder)   Congenital lobar emphysema
In 10% of CLE cases, there may be other associated malformations like congenital heart disease.
http://dx.doi.org/10.1136/bcr-2017-222290 

**Your nano-attribution (ORCID)**
lmu: chrapp, @mgriese, @kknoflach

**Comment by pnrobinson** [comment]
Are these the proposed parents? Emphysema HP:0002097; Abnormal lung lobation HP:0002101
@chrapp, @mgriese, @kknoflach

**Comment by GrieseAG** [comment]
yes, you are right, i changed it. Sorry for the mistake

**Comment by pnrobinson** [comment]
I have revised the definition since this seems to be a congenital (only) malformation:

A congenital malformation characterized by an overdistended segment of lung, affecting an party of a lobe or the entire one. It results in progressive overinflation of one or more lobes.
(https://pubmed.ncbi.nlm.nih.gov/31118601/)

Please reopen if this is not true.
